sdk/playwrighttesting/tests.mgmt.yml

```diff
@@ -0,0 +1,17 @@
+trigger: none
+
+pr: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: playwrighttesting
+    Artifacts:
+      - name: azure-resourcemanager-playwrighttesting
+        groupId: com.azure.resourcemanager
+        safeName: azureresourcemanagerplaywrighttesting
+    Clouds: 'Public'
+    UseFederatedAuth: true
```

Comment:
Could you check if this is still used in other ci?

Reply:
fixed by [commit#cfdf4c7](https://github.com/Azure/azure-sdk-for-java/pull/43414/commits/cfdf4c7cc0ceac034556d2d001ba3599b07a253a)